tests/scripts/arm64_post_build.py

```diff
@@ -91,6 +90,32 @@ def copy_core_root(core_root):
 
    return new_location
 
+def copy_tests(test_location):
+   """ Copy the test directory to the current dir as "tests"
+   Args:
+      test_location (str): location of the tests directory
+   Returns:
+      copy_location (str): name of the location, for now hardcoded to tests
+                         : for backcompat in the old system
+   """
+
+   new_location = "tests"
+
+   # Delete used instances.
+   if os.path.isdir(new_location):
+      try:
+         shutil.rmtree(new_location)
+      except:
+         assert not os.path.isdir(new_location)
+
+   try:
+      shutil.copytree(test_location, new_location)
+
+   except OSError as error:
+      log("Test location not copied. Error: %s" % error)
```

Comment:
Should we be swallowing this exception? Is there any expectation that the rest of the test run will succeed if the copy fails?